src/yacfg/files.py

```diff
@@ -1,210 +1,212 @@
-# Copyright 2018 Red Hat Inc.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#    http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
 import logging
 import os
 import re
+from typing import Tuple, NoReturn
+import importlib.util
 
 from .exceptions import ProfileError, TemplateError
-from .meta import PROFILES, TEMPLATES
+from .meta import NAME
 
-LOG = logging.getLogger(__name__)
+LOG = logging.getLogger(NAME)
 
 REX_TEMPLATE_TO_OUTPUT = re.compile(r"^(.*)(\.jinja2)$")
 
 
-def get_module_path():
-    """Get module installation path,
+def get_module_path() -> str:
+    """
+    Get module installation path,
     for reading packaged template sets and profiles
 
     :return: module installation path
     :rtype: str
```

Comment:
Duplicating the type annotation in a type comment is poor practice. I hope subsequent commits fix this!

Reply:
Yes, you're right. I should go through everything. I'm really excited about it. :grin:

Reply:
i wouldn't block this PR on it